keploy/keploy.go

```diff
@@ -702,19 +827,44 @@ func (k *Keploy) denoise(id string, tcs regression.TestCaseReq) {
 	}
 
 	// send de-noise request to server
-	r, err := http.NewRequest("POST", k.cfg.Server.URL+"/regression/denoise", bytes.NewBuffer(bin2))
-	if err != nil {
-		k.Log.Error("failed to create de-noise request", zap.String("url", tcs.URI), zap.Error(err))
-		return
-	}
-	k.setKey(r)
-	r.Header.Set("Content-Type", "application/json")
+	if k.cfg.Server.GrpcEnabled {
+		_, err = k.grpcClient.DeNoise(k.Ctx, &proto.TestReq{
```

Comment:
Here, also it should have proto.TestReq.GrpcResp field assigned when tcs is of type GRPC_EXPORT.

Reply:
Done